frontend/src/pages/ActivityReport/Pages/Review/Submitter/Draft.js

```diff
@@ -25,21 +25,21 @@ const SubmitterReviewPage = ({
   return (
     <>
       {submitted
-  && (
-  <Alert noIcon className="margin-y-4" type="success">
-    <b>Success</b>
-    <br />
-    This report was successfully submitted for approval
-  </Alert>
-  )}
+      && (
+      <Alert noIcon className="margin-y-4" type="success">
+        <b>Success</b>
+        <br />
+        This report was successfully submitted for approval
```

Comment:
Should this disable the submit button? As it is, you can make changes and submit a submitted report or even an approved report. Is that outside of the scope of this?

Reply:
Out of scope for this ticket. From the conversation on [slack](https://gsa-tts.slack.com/archives/C017RJ4GEET/p1612543608107300) we will probably have a button to explicitly "un-submit" the report on this page. I still need to create a ticket for the new work and will link once it is made.

Reply:
https://github.com/HHS/Head-Start-TTADP/issues/285